source/clients/common/microkernel_architecture/external_modules/hello_world/HelloWorldFunctionality.cpp

```diff
@@ -0,0 +1,33 @@
+#include "HelloWorldFunctionality.hpp"
+#include "MainController.hpp"
+#include "InstructionBuffer.hpp"
+#include <QDebug>
```

Comment:
We already talk about that, for me project includes should be before system ones
This is not a norm fault
Plus, there is no *official* c++ include order
Look at this question http://stackoverflow.com/questions/614302/c-header-order
The validated answer explain pretty well my way of thinking
However, here is what's the community was thinking about this question `closed as primarily opinion-based `